packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -431,6 +431,12 @@ exports.questions = {
 		question: 'Does a Tree Preservation Order (TPO) apply to any part of the appeal site?',
 		fieldName: 'tree-preservation-order',
 		validators: [new RequiredValidator()]
+	}),
+	treePreservationPlanUpload: new MultiFileUploadQuestion({
+		title: 'Tree Preservation Order extent',
+		question: 'Upload a plan showing the extent of the order',
+		fieldName: 'tree-order-upload',
+		validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]
```

Comment:
The error message here should be 'Select a plan showing the extent of the order'. Unfortunately the ticket is not yet updated

Reply:
Copy deck link is: https://pins-ds.atlassian.net.mcas.ms/wiki/spaces/AAPDS/pages/1295417526/CD17+-+Upload+a+plan+showing+the+extent+of+the+order+v13

Reply:
Updated now